aws/resource_aws_iam_user_policy.go

```diff
@@ -106,13 +116,22 @@ func resourceAwsIamUserPolicyRead(d *schema.ResourceData, meta interface{}) erro
 	if err != nil {
 		return err
 	}
-	return d.Set("policy", policy)
+	if err := d.Set("policy", policy); err != nil {
```

Comment:
Minor nitpick for the future: we do not need to perform error handling of `d.Set()` for simple `string` attributes 👍